src/multi_queue_subscriber.ts

```diff
@@ -0,0 +1,89 @@
+import { AckMessage, JobsMessage, ListenPerform, QueueJob } from "./types.ts";
+import { messagesStream } from "./ws-event-generator.ts";
+
+export class MultiQueueSubscriber<T extends Record<string, unknown>> {
```

Comment:
就叫 queue 吧，同时支持单个和多个 queue 订阅

```suggestion
export class QueueSubscriber<T extends Record<string, unknown>> {
```